<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreChangerLimiteDette</class>
 <widget class="QDialog" name="FenetreChangerLimiteDette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>126</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Raavi</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>KFet - Limite de dette</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Limite de dette pour un client :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxClient">
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Limite de dette pour un KFtier : </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxKFtier">
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="boutonOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boutonAnnuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boutonAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>FenetreChangerLimiteDette</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
